src/components/layout/InsertionLayout.stories.mdx

```diff
@@ -0,0 +1,57 @@
+import { ArgsTable, Canvas, Meta, Story } from '@storybook/addon-docs';
+
+import { Box, Center } from '@chakra-ui/react';
+
+import InsertionLayout from './InsertionLayout.tsx';
+import { FullHeight } from '../index';
+
+<Meta
+  title="Layout/Pages/Insertion"
+  component={InsertionLayout}
+  parameters={{
+    layout: 'fullscreen',
+  }}
+  args={{}}
+  argTypes={{}}
+/>
+
+export const WithContent = (args) => (
+  <FullHeight>
+    <InsertionLayout {...args}>
```

Comment:
Why do we need `args` here?

Reply:
I had some trouble with MDX 😅  will drop it thanks for the reminder